### PR TITLE
Uac2 fixes

### PR DESCRIPTION
**Pull request by twam**
Minor fixes and addition of the feature unit descriptor for UAC2

### CODE REVIEW

**Comment by twam** [comment]
Any thoughts on this? 😢

**Comment by straithe** [comment]
Thanks for the reminder! My team is stretched quite thin at the moment. We do know this is here and we really, really appreciate that you've put in a PR. I will see what I can do to get a review soon.

File: usb_protocol/types/descriptors/uac2.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
 from .standard import StandardDescriptorNumbers
 from ..descriptor import \
-    DescriptorField, DescriptorNumber, DescriptorFormat
+    DescriptorField, DescriptorNumber, DescriptorFormat, DescriptorLength
```
Comment (miek) [inline usb_protocol/types/descriptors/uac2.py:22]: This is unused

File: usb_protocol/types/descriptors/uac2.py [hunk 2]
```diff
@@ -633,9 +633,9 @@ class SidebandProtocols(IntEnum):
     "bCSourceID"          / DescriptorField(description="ID of the clock which is connected to this terminal"),
     "bNrChannels"         / DescriptorField(description="number of logical output channels in the terminal’s output channel cluster"),
     "bmChannelConfig"     / DescriptorField(description="describes the spatial location of the logical channels", default=0, length=4),
-    "bmControls"          / DescriptorField(description="OR combination of  ClockFrequencyControl, CopyProtectControl, ConnectorControl, ClusterControl, UnderflowControl and OverflowControl", default=0, length=2),
     "iChannelNames"       / DescriptorField(description="string descriptor index of the first logical channel name", default=0),
-    "iTerminal"           / DescriptorField(description="ID of the input terminal string description", default=0)
+    "bmControls"          / DescriptorField(description="OR combination of  ClockFrequencyControl, CopyProtectControl, ConnectorControl, ClusterControl, UnderflowControl and OverflowControl", default=0, length=2),
```
Comment (miek) [inline usb_protocol/types/descriptors/uac2.py:637]: Not your fault, but I think this description is incorrect? I think it should have OverloadControl and not ClockFrequencyControl.

**Review by miek — COMMENTED** [review]
This looks great, thanks! I added a couple of comments - I can fix the descriptions after if you don't want to, but just want to check that my reading of the spec is correct.

**Comment by twam** [comment]
Thanks for your feedback. Fixed the points.

**Pull request merged**